Assignment/Program.cs

```diff
@@ -1,7 +1,19 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿int reverse = 0;
+while (true) {
+   Console.Write ("Enter number :");
+   if (int.TryParse (Console.ReadLine (), out int originalNumber) && originalNumber > 0) {
+      int number = originalNumber;
+      while (number > 0) {
+         int remainder = number % 10;
+         reverse = reverse * 10 + remainder;

```

Comment:
Could have avoided the variable remainder and used the computation directly on reverse.

Reply:
ok